rewrite-java/src/main/java/org/openrewrite/java/search/SemanticallyEqual.java

```diff
@@ -929,7 +932,6 @@ public J.MemberReference visitMemberReference(J.MemberReference memberRef, J j)
                     return memberRef;
                 }
 
-                visit(memberRef.getContaining(), compareTo.getContaining());
                 visitList(memberRef.getTypeParameters(), compareTo.getTypeParameters());
```

Comment:
Do we need to revert this as well? Or can we keep this part in?
```suggestion
                visit(memberRef.getContaining(), compareTo.getContaining());
                visitList(memberRef.getTypeParameters(), compareTo.getTypeParameters());
```

Reply:
Thought about this line, it made no difference in the tests, and I decided to stick with the _brute force_ revert approach rather than backporting in this state.